src/pages/resultsView/cnSegments/CNSegments.tsx

```diff
@@ -156,8 +157,9 @@ export default class CNSegments extends React.Component<{ store: ResultsViewPage
                         onRenderingStart={this.onIgvRenderingStart}
                         onRenderingComplete={this.onIgvRenderingComplete}
                         disableSearch={this.activeLocus !== WHOLE_GENOME}
+                        isVisible={this.props.store.tabId === ResultsViewTab.CN_SEGMENTS && !this.isHidden}
```

Comment:
Component needs to be aware of its own visibility in order to prevent rendering when hidden